.projenrc.ts

```diff
@@ -374,11 +375,24 @@ new JsiiBuild(cloudAssemblySchema, {
 (() => {
   cloudAssemblySchema.preCompileTask.exec('tsx projenrc/update.ts');
 
+  // This file will be generated at release time. It needs to be gitignored or it will
+  // fail projen's "no tamper" check, which means it must also be generated every build time.
+  cloudAssemblySchema.preCompileTask.exec('[[ -f cli-version.json ]] || echo \'{ "version": "" }\' > cli-version.json');
```

Comment:
Can we not call the same `copy-cli-version-to-assembly.task.ts` script and have it create the empty file in case the version is 0.0.0? Just trying to minimize bash.